ext/spl/spl_observer.c

```diff
@@ -1301,6 +1310,7 @@ static const zend_function_entry spl_funcs_MultipleIterator[] = {
 	SPL_ME(MultipleIterator,  getFlags,               arginfo_splobject_void,                     0)
 	SPL_ME(MultipleIterator,  setFlags,               arginfo_MultipleIterator_setflags,          0)
 	SPL_ME(MultipleIterator,  attachIterator,         arginfo_MultipleIterator_attachIterator,    0)
+	SPL_ME(MultipleIterator,  __debugInfo,            arginfo_splobject_void,                     0)
```

Comment:
You've only added this to MultipleIterator, not SplObjectStorage.

Reply:
I see! Thanks.